src/Core/AdminConsole/Utilities/IntegrationTemplateProcessor.cs

```diff
@@ -20,4 +20,27 @@ public static string ReplaceTokens(string template, object values)
             return property?.GetValue(values)?.ToString() ?? match.Value;
         });
     }
+
+    public static bool TemplateRequiresUser(string template)
+    {
+        if (string.IsNullOrEmpty(template)) return false;
```

Comment:
⛏️ Control blocks should always use curly braces (even 1-line braces). Same for the other blocks below
[Reference](https://contributing.bitwarden.com/contributing/code-style/csharp#control-blocks)